tools/tools.go

```diff
@@ -0,0 +1,15 @@
+//go:build generate
+
+package tools
+
+import (
+	// document generation
+	_ "github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs"
+)
+
+// Format Terraform code for use in documentation.
+// If you do not have Terraform installed, you can remove the formatting command, but it is suggested
+// to ensure the documentation is formatted properly.
+//go:generate terraform fmt -recursive ../examples/
+// Generate documentation.
+//go:generate go run github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs generate --provider-dir ..
```

Comment:
this file is _mostly_ copy pasta of: https://github.com/hashicorp/terraform-provider-random/blob/main/tools/tools.go